spec/controllers/two_factor_authentication/otp_verification_controller_spec.rb

```diff
@@ -49,8 +49,9 @@
         sign_in_before_2fa
 
         properties = {
-          context: 'authentication',
           success?: false,
+          confirmation_for_phone_change: false,
+          context: 'authentication',
```

Comment:
I'm not opposed, but can we do that in a separate PR since this wasn't introduced here?